org.gnome.Photos.json

```diff
@@ -294,13 +294,14 @@
                 "-Dminer_fs=true",
                 "-Dminer_fs_cache_location=$XDG_CACHE_HOME/gnome-photos/miner/files",
                 "-Dminer_rss=false",
+                "-Dbattery_detection=none",
                 "-Dsystemd_user_services=false"
             ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/tracker-miners/3.4/tracker-miners-3.4.3.tar.xz",
-                    "sha256": "8e4f3976472641b648d0f8f20debae1b1629ca5b560b8616e117d29d756fbeeb",
+                    "url": "https://download.gnome.org/sources/tracker-miners/3.7/tracker-miners-3.7.0.tar.xz",
```

Comment:
Note: the gnome f-e-d-c want to downgrade because odd version number.

Reply:
stable-only: false. I should go back to the checker and have a test for tracker-miners

Reply:
Or just stop pretending there are modules releasing with the old 3.XYZ versioning scheme.

Reply:
Added `stable-only: false` and f-e-d-c no longer complains.